record_web/lib/record_web.dart

```diff
@@ -20,6 +22,10 @@ class RecordPluginWeb extends RecordPlatform {
   // Completer to get data & stop events before `stop()` method ends
   Completer<String>? _onStopCompleter;
   StreamController<RecordState>? _stateStreamCtrl;
+  double maxAmplitude = -160;
```

Comment:
We should add two const for max and min. Theorically, min should be -160 and max 0

Reply:
yes but this is meant to be used as max apmlitude not the default
even after adding two defaults we still need this

Reply:
I agree, I just want to avoid multiple scalar values in the class with two new constants.

Reply:
done